diff --git a/node_modules/formidable/src/Formidable.js b/node_modules/formidable/src/Formidable.js
index 0542700..72191fa 100644
--- a/node_modules/formidable/src/Formidable.js
+++ b/node_modules/formidable/src/Formidable.js
@@ -5,7 +5,7 @@
 
 const os = require('os');
 const path = require('path');
-const hexoid = require('hexoid');
+const { hexoid } = require('hexoid');
 const once = require('once');
 const dezalgo = require('dezalgo');
 const { EventEmitter } = require('events');
